### PR TITLE
update to doc to make copy and past easier and to add more info

### PR DESCRIPTION
**Pull request by R-Lawton**

### CODE REVIEW

File: docs/gateways/define-and-place-a-gateway.md [hunk 1]
```diff
@@ -3,91 +3,151 @@
 In this guide, we will go through defining a Gateway in the OCM hub cluster that can then be distributed to and instantiated on a set of managed spoke clusters.
 
 ### Pre Requisites
+* Complete the [Getting Started Guide](https://docs.kuadrant.io/getting-started/) to bring up a suitable environment. Going through the installation will ensure that a supported `GatewayClass` is registered in the hub cluster that the Kuadrant multi-cluster gateway controller will handle. 
 
-- Go through the [getting started guide](https://docs.kuadrant.io/getting-started/). 
+If you are looking to change provider from the default Istio:
+* Please have the Gateway provider of your choice installed and configured (in this example we use Envoy gateway see the following [docs](https://gateway.envoyproxy.io/v0.5.0/user/quickstart.html))
```
Comment (sergioifg94) [inline docs/gateways/define-and-place-a-gateway.md:9]:
```suggestion
* Please have the Gateway provider of your choice installed and configured (in this example we use Envoy gateway. See the following [docs](https://gateway.envoyproxy.io/v0.5.0/user/quickstart.html))
```

File: docs/gateways/define-and-place-a-gateway.md [hunk 2]
```diff
@@ -3,91 +3,151 @@
...
+
+Set the kubecontext for each terminal, refer back to these commands if re-config is needed.
+
+In `T1` run `kind export kubeconfig --name=mgc-control-plane --kubeconfig=$(pwd)/control-plane.yaml && export KUBECONFIG=$(pwd)/control-plane.yaml`
+
+In `T2` run `kind export kubeconfig --name=mgc-workload-1 --kubeconfig=$(pwd)/workload1.yaml && export KUBECONFIG=$(pwd)/workload1.yaml`
+
+export `MGC_SUB_DOMAIN` in each terminal if you haven't already added it to your `.zshrc` or `.bash_profile`.
+
+**NOTE** :exclamation: The quick start script will create a placement resource as part of the setup you can use this as further inspiration for other placement resources you would like to create.
```
Comment (sergioifg94) [inline docs/gateways/define-and-place-a-gateway.md:29]:
```suggestion
**NOTE** :exclamation: The quick start script will create a placement resource as part of the setup. You can use this as further inspiration for other placement resources you would like to create.
```

**Review by sergioifg94 — COMMENTED** [review]
Looks good, just a couple of writing suggestions

**Comment by jasonmadigan** [comment]
👀

File: docs/gateways/define-and-place-a-gateway.md [hunk 3]
```diff
@@ -2,92 +2,142 @@
...
-metadata:
-  name: gateway-params
-  namespace: multi-cluster-gateways
-```
-
-Once this has been created, any gateway created from that gateway class will result in a downstream gateway being provisioned with the configured downstreamClass.
\ No newline at end of file
+If you are only using the DNSPolicy and TLSPolicy resources, you can use these APIs with any Gateway provider. To change the underlying provider, you need to set the gatewayclass param `downstreamClass`. 
+
+1.  Create the following configmap (eg stands for envoy gateway see above prerequisites):
```
Comment (jasonmadigan) [inline docs/gateways/define-and-place-a-gateway.md:113]:
```suggestion
1.  Create the following configmap. Note: In this example, 'eg' stands for the Envoy gateway, which is mentioned in the prerequisites above:
```

**Review by jasonmadigan — APPROVED** [review]

File: docs/gateways/define-and-place-a-gateway.md [hunk 4]
```diff
@@ -2,92 +2,142 @@
...
-### Pre Requisites
+### Prerequisites
+* Complete the [Getting Started Guide](https://docs.kuadrant.io/getting-started/) to bring up a suitable environment. 
 
-- Go through the [getting started guide](https://docs.kuadrant.io/getting-started/). 
+If you are looking to change provider from the default Istio:
+* Please have the Gateway provider of your choice installed and configured (in this example we use Envoy gateway. See the following [docs](https://gateway.envoyproxy.io/v0.5.0/user/quickstart.html))
 
-You should start this guide with OCM installed, 1 or more spoke clusters registered with the hub and Kuadrant installed into the hub.
+## Inital setup
```
Comment (jasonmadigan) [inline docs/gateways/define-and-place-a-gateway.md:11]:
```suggestion
## Initial setup 
```

**Comment by jasonmadigan** [comment]
Oops, just one more fix @R-Lawton

**Pull request merged**